tensordict/utils.py

```diff
@@ -1851,6 +1851,85 @@ def format_size(size):
         logging.info(indent + os.path.basename(path))
 
 
+def isin(
+    tensordict: TensorDictBase,
+    reference_tensordict: TensorDictBase,
+    key: str,
+    dim: int = 0,
+    invert: bool = False,
+):
+    """Tests if each element of 'key' in tensordict 'dim' is also present in the reference_tensordict.
+
+    This function returns a boolean tensor of length  ´tensordict.batch_size[dim]´ that is True for elements in
+    'key' that are also present in the reference_tensordict. This function assumes that both tensordict and
+    reference_tensordict have the same batch size and contain the specified key, otherwise it will raise an error.
+
+    Args:
+        tensordict (TensorDictBase): Input TensorDict.
+        reference_tensordict (TensorDictBase): Target TensorDict against which to test.
+        key (str): The key to test.
+        dim (int): The dimension along which to test. Default: 0
+        invert (bool):  If True, inverts the boolean return tensor, resulting in True values for elements
+            not in reference_tensordict. Default: False
```

Comment:
```suggestion
    Args:
        input (TensorDictBase): Input TensorDict.
        reference (TensorDictBase): Target TensorDict to compare to ``input``.
        key (NestedKey): The key to test.
        dim (int, optional): The dimension along which to test. Defaults to ``0``.
    
    Keyword Args:
        invert (bool, optional):  If ``True``, inverts the boolean return tensor, resulting in ``True`` values for elements
            not in ``reference``. Defaults to ``False``.
```

To me `invert` brings very little and should be removed. You could just to `~isin(td0, td1)` and it would give the same result for just 1 extra character. I tend to think about kwargs as bringing non-trivial functionalities.